Assets/MRTK/Examples/Demos/Input/Scenes/PrimaryPointer/PrimaryPointerHandlerExample.cs

```diff
@@ -24,7 +24,19 @@ private void OnPrimaryPointerChanged(IMixedRealityPointer oldPointer, IMixedReal
             {
                 if (newPointer != null)
                 {
-                    Transform parentTransform = newPointer.BaseCursor.TryGetMonoBehaviour(out MonoBehaviour baseCursor) ? baseCursor.transform : null;
+                    Transform parentTransform = null;
+
+                    // For this example scene, use special logic if a GGVPointer becomes the primary pointer. 
+                    // In particular, the GGV pointer defers it's cursor management to the GazeProvider, which has it's own internal pointer definition as well
+                    // In the future, the pointer/cursor relationship will be reworked and standardized to remove this awkward set of co-dependencies
+                    if (newPointer is GGVPointer)
+                    {
+                        parentTransform = CoreServices.InputSystem.GazeProvider.GazePointer.BaseCursor.TryGetMonoBehaviour(out MonoBehaviour baseCursor) ? baseCursor.transform : null;
```

Comment:
To make sure I understand...the GGVPointer _has_ a cursor reference, it's just not correct? Or the GGVPointer doesn't have a `BaseCursor` set at all?

Reply:
The GGV pointer doesn't have BaseCursor set at all, and is not designed to have one set, since then the cursor state's management gets muddied between the GGVPointer and the GazeProvider.

Reply:
Thanks!